packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -68,11 +63,136 @@ $usa-banner-text-color: #1b1b1b;
     padding-right: 2px;
     width: 0.5rem;
   }
+
+  // Hide header-action text for large viewport and when expanded
+  @media (min-width: $width-sm) {
+    display: none;
+  }
+  .ds-c-usa-banner__header--expanded & {
+    display: none;
+  }
+}
+
+.ds-c-usa-banner__content {
+  font-size: 0.99816rem;
+  margin-left: auto;
+  margin-right: auto;
+  overflow: hidden;
+  padding: 0.25rem 1rem 1rem 0.5rem;
+
+  @media (min-width: $width-sm) {
+    padding-bottom: 1.5rem;
+    padding-top: 1.5rem;
+  }
+}
+
+.ds-c-usa-banner__button {
+  background-color: transparent;
+  border: 0;
+  border-radius: 0;
+  bottom: 0;
+  box-shadow: none;
+  color: #005ea2;
+  display: block;
+  font-size: 0.8rem;
+  -moz-osx-font-smoothing: auto;
```

Comment:
What is this from? Is this copied from the USWDS CSS? If so, it would be better to reference the SCSS

Reply:
This SCSS contains alot of random values like `#005ea2`. We should be using variables for commonly used variables so they are easier to understand and modify.

Reply:
Thanks for for picking this up. Will refactor to use variables to make it more understandable and customizable.